src/sidebar/components/test/toast-messages-test.js

```diff
@@ -0,0 +1,146 @@
+import { mount } from 'enzyme';
+import { createElement } from 'preact';
+import { act } from 'preact/test-utils';
+
+import mockImportedComponents from '../../../test-util/mock-imported-components';
+
+import ToastMessages, { $imports } from '../toast-messages';
+//import { checkAccessibility } from '../../../test-util/accessibility';
```

Comment:
can be removed

Reply:
Actually, I'd argue instead that I forgot to add `a11y` tests, which I'll do now! I wonder if there's something automated that could catch when we forget to add these tests.

Reply:
...and they kicked up something legit, so, yay!